.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+# Description
+
+Please include a summary of changes, motivation and context for this PR.
+
+Fixes # (issue)
+
+## Type of change
+
+Please tick all options which are relevant.
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Requires documentation updates
+
+# Testing
+
+Please describe the test(s) that you added and ran to verify your changes.
+
+## Test(s) description
+
+**Test Configuration**:
+
+* OS:
+
+# Checklist:
+
+- [ ] I have added tests that prove my fix is effective and/or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] I have used understandable variable names
+- [ ] I have squashed my commits into a single commit ready for review
+- [ ] I have run `git-clang-format HEAD~` on my changes
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas

```

Comment:
Can we collapse these bullets into on that says something similar to “I have read the contribution guide recently” and move these there?

Reply:
and maybe provide a guide to open the PR using labels in the name like:

[ci/Interpreter/docs/tests...] title

Fixes # (issue)

- [ ] Tested locally 

Test configuration:

Reply:
Yeah.

Reply:
> Can we collapse these bullets into on that says something similar to “I have read the contribution guide recently” and move these there?

@vgvassilev Do we have a contributors guide somewhere already, and I just can't find it, or do I need to make one?

Reply:
I think we need to make one. Look at the one in Clad.